sdk-core/assembly/index.ts

```diff
@@ -9,5 +9,8 @@ export * from "./logging";
 export * from "./math";
 export * from "./promise";
 export * from "./datetime";
+import * as JSON from "@serial-as/json";
+export { JSON };
+export * from "@serial-as/borsh";
```

Comment:
What are the implications of this change? I didn't see it mentioned in any updated docs or in the PR description (do we use the PR description to create release notes? might be useful)

Reply:
In the future branch this isn't needed because we have a `serializer.ts` which exports a simplified serializer.  So I think I'll add that to this PR.  And add to PR description.  I also think we should make a CHANGELOG.md to formalize this.